### PR TITLE
adding a variable to control the api gateway integration timeout

### PR DESCRIPTION
**Pull request by mb2con**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -49,7 +49,10 @@ variable "api_gateway_throttling_rate_limit" {
   description = "APIGateway throttling rate limit, default is -1 which does not enforce a limit"
   default     = -1
 }
-
+variable "api_gateway_timeout_milliseconds" {
+  description = "APIGateway integration request timeout (in milliseconds)"
+  default     = 6000
```
Comment (pmcgrath) [inline variables.tf]: @mb2con  Would leave this default value at 29000, that way this is an override and all existing tenants should remain as they are, see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_integration#timeout_milliseconds

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file.
 
+## ## [v0.1.10] Unreleased
+- Added: Added "api_gateway_timeout_milliseconds" variable to control the API Gateway
```
Comment (pmcgrath) [inline CHANGELOG.md]: Minor, but probably no need for the "Added: Added" - can use "Added: "
Probably have not been consistent in previous entries we have added

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file.
 
+## ## [v0.1.10] Unreleased
```
Comment (pmcgrath) [inline CHANGELOG.md]: Can we remove the extra ## - only need one of them
Would also remove the [v0.1.10] - what if we were to go to 0.2.0 - even though we do not use semantic versioning and not sure we will ever

File: variables.tf (hunk 1 above)
Comment (mb2con) [reply variables.tf]: changed to 29000

File: CHANGELOG.md (hunk 2 above)
Comment (mb2con) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (mb2con) [reply CHANGELOG.md]: removed.

**Review by pmcgrath — APPROVED** [review]
LGTM

**Pull request merged**